doc/driver.md

```diff
@@ -224,6 +235,11 @@ let odoc_libraries = [
     "odoc_model_desc"; "odoc_model"; "odoc_manpage"; "odoc_loader";
     "odoc_latex"; "odoc_html"; "odoc_document"; "odoc_examples" ];;
 
+let source_dir_of_odoc_lib lib =
+  match String.split_on_char '_' lib with
+  | "odoc" :: s -> Some Fpath.(v ".." / "src" / String.concat "_" s)
```

Comment:
That's not the ideal approach because it won't work on external dependencies.

It would be great if the reference driver could use `ocamlfind` for Odoc itself too. This would allow to write a general way to find the source files. (which `source_files_of_odoc_module` is already doing)

Calling `ocamlfind query` on Odoc's libraries should return something in the `_build/install` directory. That's the better things to build the doc from.

Reply:
I agree that it would be better, however all my attempts at using `ocamlfind` (which I don't know well) for `odoc` resulted in `Not found` answers. I can try to investigate a bit more.

I think that it is good to keep the driver simple, and not that bad to show that implementation and interface are optionals.

Reply:
`ocamlfind` is working as expected from within the driver:

```
$ ocamlfind query odoc
/home/juloo/w/odoc/_build/install/default/lib/odoc
```

This would allow to treat odoc as any other package and to implement a general function for finding the sources. I think such a function would be more interesting to the reader.

It wouldn't make the driver more complex but it's definitely a lot of work (basically rewrite the entire driver).
Let's not do this.